pkg/scheduler/framework/runtime/framework.go

```diff
@@ -126,12 +126,13 @@ func (f *frameworkImpl) getExtensionPoints(plugins *config.Plugins) []extensionP
 		{plugins.PostFilter, &f.postFilterPlugins},
 		{plugins.Reserve, &f.reservePlugins},
 		{plugins.PreScore, &f.preScorePlugins},
-		{plugins.Score, &f.scorePlugins},
 		{plugins.PreBind, &f.preBindPlugins},
 		{plugins.Bind, &f.bindPlugins},
 		{plugins.PostBind, &f.postBindPlugins},
 		{plugins.Permit, &f.permitPlugins},
 		{plugins.QueueSort, &f.queueSortPlugins},
+		// Keep plugins.Score at the end for weighting.
+		{plugins.Score, &f.scorePlugins},
```

Comment:
I haven't dug too deep into it, but this seems like it misses addressing the actual cause of this bug from your description in the linked issue. Perhaps we can refactor to actually address it? Or maybe the work necessary isn't worth the tradeoff

Reply:
Thx. I updated this issue. IMO, this small change fixed this issue by less effort.

Reply:
@damemi  current IMPL doesn't differentiate between different plugins, but instead only keep one plugin instance in the `map`, which mean the latter will overridden the former one as long as the plugin name is same.

+1 this should be the easiest fixing, but I am not sure whether there is any better approach.